lib/saluki-components/src/transforms/host_tags/mod.rs

```diff
@@ -124,3 +129,44 @@ impl SynchronousTransform for HostTagsEnrichment {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+
+    use std::{
+        num::NonZeroUsize,
+        time::{Duration, Instant},
+    };
+
+    use bytesize::ByteSize;
+    use saluki_context::{Context, ContextResolverBuilder};
+    use saluki_event::metric::Metric;
+
+    use super::*;
+
+    #[tokio::test]
+    async fn test_host_tags_enrichment() {
```

Comment:
We try to avoid stuttering, so no `test_` prefix in the function name... and maybe just call it something like `basic`?